docs/proposals/branch-and-release/README.md

```diff
@@ -0,0 +1,167 @@
+
+## Evergreen Branches
+
+### main
+
+The `main` branch tracks what's currently deployed at
+<https://www.enterprisedb.com/docs/>.
+
+### develop
+
+For the most part, `develop` tracks the next release of the `docs` repo.
+`develop` is deployed automatically at
+<https://edb-docs-staging.netlify.app/docs/>.
+
+The `develop` branch should always be in a deployable state.
+
+## Release Process
+
+### Preparation
+
+1. Make a release branch from `develop`: `release/${today's date}`. E.g.
+   `release/2021-02-09`.
+1. Create a pull request ("PR") to merge the release branch into `main`.
+1. Smoke test the release branch and make adjustments as needed. Ideally you
+   shouldn't need to make any new commits to the release branch.
+
+### Release
+
+1. Merge the release PR.
+1. Ensure the release is successful.
+1. Delete the release branch.
+1. Tag `main` with the release date: e.g. `2021-02-09`.
+
+   If there are multiple releases per day, suffix with letters. For example,
+   the second release of the day would be `2021-02-09-a`, third release would
+   be `2021-02-09-b`, and so on.
+
+1. Merge `main` into `develop`.
+
+## Branching Strategies
+
+Example: Create a `bug/epas/fix-typo` branch from `develop` ("base").
+
+See [Use Cases](#use-cases) for when to use which strategy.
+
+### Short-lived
+
+Ideally, most branches are short-lived from creation to merging into its base
+branch. This minimizes potential conflicts and typically allows the branch to
+be merged without any fuss.
+
+### Long-lived
+
+In the case of long-lived branches (e.g. future product release branches), it's
+up to the person responsible for the branch to keep it up to date with its base
+branch. There are couple ways to periodically keep your branch up-to-date (once
+a day, or less frequently if the base branch is stable):
+
+-  Rebase your branch onto the base branch (preferred).
+-  Merge the base branch into your branch.
+
+This will ensure that conflicts and surprises are resolved as early in the
+branch lifecycle as possible.
+
+## Pull Request Strategies
+
+Each branch that's intended to be deployed at some point needs to be associated
+with a PR. If there's a ticket associated with the PR, provide a link to the
+ticket at the top of the PR description.
+
+See [Use Cases](#use-cases) for when to use which strategy.
+
+### Normal PR
+
+A normal PR is one that's ready to be reviewed and merged to the base branch.
+It requires at least one approval in order to be merged.
+
+Once a PR is approved, anyone can merge it to the base branch.
+
+### Draft PR
+
+A [draft PR][pr-draft] is _not_ ready to be merged to the base branch. However, it may
+still be a good idea to solicit work-in-progress reviews.
+
+Once the PR is ready, [remove the "Draft" status and convert it to a normal
+PR][pr state].
+
+## Use Cases
+
+### Hot Fix
+
+Stop the presses, something is seriously wrong with
+<https://www.enterprisedb.com/docs/>.
+
+-  **Base branch**: `main`
```

Comment:
Is this for the situation where there is somehow an issue that only affects `main`, but not `develop`? If that's not the case, with this strategy `develop` would be missing the hot fix, correct?

Reply:
This is meant to address the case of a production emergency case, when we're not ready to release `develop` yet.

<https://github.com/EnterpriseDB/docs/pull/915/files#diff-47f702b951edf20e1664629415407877c1f90a3e198c871989021b8bd707e2b5R59> ensures that hot fix changes make it back to `develop`.